elfi/distributions.py

```diff
@@ -0,0 +1,334 @@
+"""This module implements empirical estimations of distributions.
+
+Currently it is mainly used for Gaussian copula ABC.
+
+References
+----------
+Jingjing Li, David J. Nott, Yanan Fan, Scott A. Sisson
+Extending approximate Bayesian computation methods to high dimensions via Gaussian copula.
+Computational Statistics & Data Analysis
+Volume 106, February 2017, Pages 77-89
+https://doi.org/10.1016/j.csda.2016.07.005
+
+"""
+import numpy as np
+import scipy.stats as ss
+from scipy.interpolate import interp1d
+
+__all__ = ('EmpiricalDensity', 'ecdf', 'eppf', 'estimate_densities', 'MetaGaussian')
+
+
+def ecdf(samples):
```

Comment:
I'm confused if/how these are related to the EmpiricalDensity, and why are they not implemented as its methods?

Reply:
They are not related. They are separate functions that I though would be useful.